index.html

```diff
@@ -25,7 +25,30 @@ <h2>Here are your results:</h2>
   <div id="add-js">
 
   </div>
+  
   <script src="js/script.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.1/lib/darkmode-js.min.js"></script>
+  <script>
+  //This uses the cdn above and calls Dark mode and shows widget
+  new Darkmode().showWidget();
+  //These are all you widget options
+  var options = {
+  bottom: '64px', // default: '32px'
+  right: 'unset', // default: '32px'
+  left: '32px', // default: 'unset'
+  time: '0.5s', // default: '0.3s'
+  mixColor: '#fff', // default: '#fff'
+  backgroundColor: '#fff',  // default: '#fff'
+  buttonColorDark: '#100f2c',  // default: '#100f2c'
+  buttonColorLight: '#fff', // default: '#fff'
+  saveInCookies: false, // default: true,
+  label: '🌓', // default: ''
+  autoMatchOsTheme: true // default: true
+  }
```

Comment:
@wajeehsheikh : Nicely done. These changes are are just superficial, but it would make the code cleaner:
1) remove the inline comments as they are unnecessary and add no value to the `options` object.
2) indent the `options` object by 2 spaces:
```js
var options = {
  bottom: '64px',
  right: 'unset',
  left: '32px',
  time: '0.5s',
  mixColor: '#fff'
  backgroundColor: '#fff',
  buttonColorDark: '#100f2c',
  buttonColorLight: '#fff',
  saveInCookies: false,
  label: '🌓',
  autoMatchOsTheme: true 
}

Reply:
I see that you have included the issue number in the title of the pull request (PR), which is great, but it would be stupendous if you start referencing the issue in the PR description as well. https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

Reply:
I agree with your comment about indentation, however, I think the default values are helpful during the pull request stage. Once the code is added to the master and is included in a stable release, I'll remove them. Thanks!